transport/tchannel/channel_outbound.go

```diff
@@ -159,7 +159,7 @@ func (o *ChannelOutbound) Call(ctx context.Context, req *transport.Request) (*tr
 	// Inject tracing system baggage
 	reqHeaders := tchannel.InjectOutboundSpan(call.Response(), req.Headers.Items())
 
-	if err := writeRequestHeaders(ctx, format, reqHeaders, call.Arg2Writer); err != nil {
+	if err := writeRequestHeaders(ctx, format, reqHeaders, call.Arg2Writer, o.transport.forwardingHeader); err != nil {
```

Comment:
`forwardingHeader` does not read right here. Should read like "use exact case" as opposed to "use canonicalized case".